modules/benchmark/R/calc.benchmark.R

```diff
@@ -11,21 +11,29 @@
 
 calc.benchmark <- function(settings, bety) {
   
-  library(RPostgreSQL)
-  library(data.table)
+  # dplyr functions
+  tbl     <- dplyr::tbl
+  filter  <- dplyr::filter
+  rename  <- dplyr::rename
+  collect <- dplyr::collect
+  select  <- dplyr::select
   
```

Comment:
if you end up using lubridate::year a lot you could put that at the top as well (i.e. year <- lubridate::year) and just use year...but that is just a preference thing of course